front/cypress/e2e/project_card.cy.ts

```diff
@@ -25,11 +24,12 @@ describe('Project card component', () => {
         canPost: true,
         canReact: true,
       });
-      cy.goToLandingPage();
     });
   });
 
   it('shows the title, description, progress bar and cta', () => {
```

Comment:
This should actually not be tested here, but because ProjectCard is such a gigantic component, it wasn't easy to test half of what's tested here in a unit test so keeping for now.